.configs/.sourcery.yaml

```
@@ -0,0 +1,83 @@
+- ~/.pyenv
+- ~/.rye
+- ~/.vscode
+- .vscode
+- ~/.cache
+- ~/.config
+- ~/.local
+- thedozens/staticfiles
+
+rule_settings:
+  enable:
+  - default
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)